perf-tests/clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -18,6 +18,10 @@ package slos
 
 import (
 	"fmt"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	"strings"
```

Comment:
perf test  failed to  start with error:
```
# k8s.io/kubernetes/perf-tests/clusterloader2/pkg/measurement/common/slos
pkg/measurement/common/slos/pod_startup_latency.go:25:2: imported and not used: "strings"
```

Reply:
Thank you sonya. This was fixed in "minor fix"